src/convenience/frameworks.deno.ts

```diff
@@ -44,6 +61,13 @@ const serveHttp = (requestEvent: Deno.RequestEvent) => ({
         requestEvent.respondWith(
             new Response(JSON.stringify(json), { status: 200 }),
         ),
+    unauthorized: () =>
+        requestEvent.respondWith(
+            new Response('"unauthorized"', {
+                status: 401,
+                statusText: "secret token is wrong",
```

Comment:
I'm not sure if statusText is meant for that. We should probably put that text into the first argument